### PR TITLE
Bl 158 update availability views

### PR DESCRIPTION
**Pull request by sensei100**
This PR updates the way availability information is displayed on the index page.
- If a record contains only one online link, clicking online should go directly to the resource.
- If a record has multiple resources, a collapse div appears with all the resource links listed
- If the item is a physical item, clicking 'checking availability' should show the 'get-it iframe'

### CODE REVIEW

File: app/views/catalog/_index_default.html.erb [hunk 1]
```diff
@@ -1,20 +1,38 @@
...
   <% end -%>
 
-  <dt class="blacklight-availability">Status/Location:</dt>
-  <dd class="blacklight-availability availability-ajax-load" data-availability-ids="<%= document.alma_availability_mms_ids.join(',') %>" >Loading...</dd>
-  <dt class="availability-show-on-ajax-load hide"></dt>
-  <dd class="availability-show-on-ajax-load hide">
-    <button class="btn btn-default availability-toggle-details" data-show-text="Show Availability Details" data-hide-text="Hide Availability Details">Show Availability Details</button>
-  </dd>
-</dl>
+  <% @response["response"][:docs].each_with_index {|field, i| %>
```
Comment (dkinzer) [inline app/views/catalog/_index_default.html.erb]: Would it be possible to move this out of the template/view?

Comment (sensei100) [reply app/views/catalog/_index_default.html.erb]: I created a Jira ticket to take a look at that in the off week.

Comment (bibliotechy) [reply app/views/catalog/_index_default.html.erb]: @sensei100 Agreed, I'd like to merge this so Emily can QA the changes. We can come back to clean it up next week.

@dkinzer Any other blockers on this PR?

Comment (dkinzer) [reply app/views/catalog/_index_default.html.erb]: I don't see anything else.  I don't know how to QA this, though ,which is why I have not merged.

Comment (bibliotechy) [reply app/views/catalog/_index_default.html.erb]: @dkinzer OK. So, for work that tends to be mostly UI, I expect code review to be mostly about reviewing the code. Emily will handle the UI QA as part of her process.

**Comment by bibliotechy** [comment]
OK, refactoring view logic, probably out to a partial,  to be addressed in BL-168, so we can get these changes into libqa to be reviewed.

**Pull request merged**